metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/parser_master.py

```diff
@@ -54,6 +57,12 @@ def configure(self, env, upgrade_type=None, config_dir=None):
             commands.init_kafka_acls()
             commands.set_acl_configured()
 
+        File(format("{metron_config_path}/pcap.properties"),
```

Comment:
I had earlier place this under the [indexing_master.py ](https://github.com/apache/metron/blob/master/metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/indexing_master.py#L62)script, but felt that it is more related to parser than with indexing. I am happy to move this block to any other place that is deemed fit.